client/Wayland/wlf_pointer.c

```diff
@@ -39,16 +39,18 @@ typedef struct wlf_pointer wlfPointer;
 static BOOL wlf_Pointer_New(rdpContext* context, rdpPointer* pointer)
 {
 	wlfPointer* ptr = (wlfPointer*)pointer;
+
 	if (!ptr)
 		return FALSE;
 
 	ptr->size = pointer->width * pointer->height * 4;
 	ptr->data = _aligned_malloc(ptr->size, 16);
+
 	if (!ptr->data)
 		return FALSE;
 
 	if (!freerdp_image_copy_from_pointer_data(
-	        ptr->data, PIXEL_FORMAT_RGBA32,
+	        ptr->data, PIXEL_FORMAT_BGRA32,
```

Comment:
Color format issue, missed in #5221 (the cursors I´ve tested with had little R and B components :/